framework/console/controllers/FixtureController.php

```diff
@@ -415,7 +415,15 @@ private function findFixtures(array $fixtures = [])
         $foundFixtures = [];
 
         foreach ($files as $fixture) {
-            $foundFixtures[] = basename($fixture, 'Fixture.php');
+            // strip $fixturesPath from $fixture's full path
+            $relativeName = str_replace($fixturesPath . "/", "", $fixture);
+            // get fixtures's directory
+            $relativeDir = dirname($relativeName) == '.' ? '' : dirname($relativeName) . '/';
+            // get fixture name relatively to $fixturesPath
+            $relativeName = $relativeDir . basename($fixture, 'Fixture.php');
+            // normalize path to unix format
+            $relativeName = str_replace("\\", "/", $relativeName);
```

Comment:
Good point. Yes, there might be a problem - `$fixture` is calculated automatically, whereas `$fixturesPath` is provided or may be provided by user. So it's not guaranteed they're in the same format.
I'll provide corresponding changes soon.